config/sequelize-cli-config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
why do we need this block?

Reply:
This is the configuration of Sequelize cli which was also generated after running `npx sequelize-cli init`, a module is used instead of [a JSON shown in the document ](https://sequelize.org/docs/v6/other-topics/migrations/#configuration)in order to all the environment variables

Reply:
1. add a comment to explain the reason for this block.
2. didn't we say to replace the DATABASE_URL with user, password, etc?